functional-tests/tests/sync/sync_from_rpc.py

```diff
@@ -62,3 +63,16 @@ def main(self, ctx: flexitest.RunContext):
             \tseq {block_from_sequencer},\n\tfn {block_from_fullnode}"
         )
         assert seq_el_hash == fn_el_hash, "EL blocks don't match"
+
+        # Check fullnode sees same checkpoint reference as sequencer
+        epoch = 1
+        wait_until(
+            lambda: seqrpc.strata_getCheckpointInfo(epoch)["confirmation_status"] != "pending",
+            error_with="Did not see checkpoint in l1",
+            timeout=15,
+        )
```

Comment:
Should have a dedicated utils fn for this.  This is rather ad-hoc.  See the others I added recently.

Reply:
Yes, used the one there's already there.